### DISCUSSION TITLE
提议移除 bundlesize 使用 size-limit 代替

### DISCUSSION THREAD

**Discussion by izhouteng** [ant-design/ant-design#36574] [Ideas]
`bundlesize` 库早已停止维护，我提议更换成 `size-limit`。

同时，`bundlesize` 依赖的 `brotli-size@0.0.3`，又依赖 `iltorb`。

而 `iltorb` 早已废弃，现在 Node.js 官方的`zlib`模块可以代替它。

更要命的是，`iltorb` 还会需要 `node-gyp` 和 `Visual Studio Desktop development with C++`（非常庞大）。

这是整个 `antd` 项目唯一需要 `node-gyp` 的地方。

仅仅为了安装一个 `bundlesize` 模块，去安装 `Visual Studio C++ 桌面开发环境` 实在不妥。

这也容易劝退想为 `antd` 贡献代码的新人。

size-limit see: [https://github.com/ai/size-limit](https://github.com/ai/size-limit)

@afc163

**Comment by afc163** [comment]
@izhouteng 来个 PR 吧，先双跑一段时间看看效果，能平替再干掉 bundlesize。

**Comment by fengmk2** [comment]
https://github.com/ant-design/ant-design/pull/36712

**Comment by fengmk2** [comment]
可以删除了，不删除的话使用它的 v2 https://github.com/ant-design/ant-design/pull/39216
